api/controllers/game/face-card.js

```diff
@@ -65,7 +65,7 @@ module.exports = function (req, res) {
         victory,
       });
       // If the game is over, clean it up
-      if (victory.gameOver) await gameService.clearGame({ userId: req.session.usr });
+      if (victory.gameOver) {await gameService.clearGame({ userId: req.session.usr });}
```

Comment:
I think these should always be on a new line.

Reply:
Modified the same in all files and updated.